fairroot/base/steer/FairRunAnaProof.cxx

```diff
@@ -238,19 +235,18 @@ void FairRunAnaProof::InitContainers()
     fRtdb = GetRuntimeDb();
     FairBaseParSet* par = dynamic_cast<FairBaseParSet*>(fRtdb->getContainer("FairBaseParSet"));
 
+    auto* evtHeader = GetEventHeader();
     if (par && fInFileIsOpen) {
-        fRootManager->SpecifyRunId();
-
-        GetEventHeader();
+        fRootManager->ReadEvent(0);
```

Comment:
I am not quite sure that `SpecifyRunId` and `ReadEvent(0)` do the same thing? I am confused.